### PR TITLE
Diagnostic Settings for Web Apps

### PR DESCRIPTION
**Pull request by Joshua-Donovan**
To enable the ability to configure all diagnostic settings for Web Apps. The problem to overcome is Premium App Service Plans have two additional diagnostic settings vs Standard App Service Plans: "AppServiceFileAuditLogs" & "AppServiceAntivirusScanAuditLogs". The App Service Plan is a different resource in Azure from the Web App we will be targeting with this policy. This policy uses a nested deployment to reference the App Service Plan's SKU to know what diagnostic settings to install on the Web App and avoid an error on Standard App Service Plans for attempting to install diagnostic settings that are not available.

### CODE REVIEW

**Comment by Joshua-Donovan** [comment]
Any maintainers able to check this? Waiting on approval.

**Review by aschabus — CHANGES_REQUESTED** [review]
Can you please remove the README.md as we decided that we are not adding readme's any longer.
Thanks
Andi

**Pull request merged**
